src/components/BaseButton.vue

```diff
@@ -7,6 +7,7 @@ interface Props {
   type?: 'primary' | 'brand' | 'danger' | 'link';
   size?: 'default' | 'small';
   variant?: 'filled' | 'outline';
+  href?: string;
```

Comment:
```suggestion
  /**
   * Transform the button into an anchor tag. If null then the component will be rendered as a button tag.
   */
  href?: string;
```

You might need to add `vue-component-meta` to dev deps for this to show inline on the storybook docs but I think it's useful to mark this.